rpmconf/Spec/Scripts/zimbra-store.pre

```diff
@@ -23,3 +23,9 @@ fi
 if [ -d "/opt/zimbra/mailboxd/work/zimbraAdmin" ]; then
   rm -rf /opt/zimbra/mailboxd/work/zimbraAdmin > /dev/null 2>&1
 fi
+
+if [ -d /opt/zimbra/lib/ext/smime ]; then
```

Comment:
we need to invest more time to investigate why this is with SMIME extension only.

Reply:
@umagmrit during every upgrade we expect the extension to get cleared and add new ones?

Reply:
yes @dasiyogesh